src/emcpy/plots/mapping.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+from emcpy.utils import roundNumber
+from emcpy.plots import domains
+
+__all__ = ['spatial']
+
+
+def _map_scatter(latitude, longitude, data, domain, plotmap, plotopts):
+    """
+    Plot 1-dimensional data as scatter.
+    """
+    fig = plt.figure(figsize=plotopts['figsize'])
+
+    if plotmap:
+        ax = domains.get_domain(fig, domain)
+        cs = plt.scatter(longitude, latitude, c=data,
+                         s=plotopts['marker size'], vmin=plotopts['vmin'],
+                         vmax=plotopts['vmax'], cmap=plotopts['cmap'],
+                         transform=ccrs.PlateCarree())
+        if plotopts['grid']:
+            ax.gridlines(crs=ccrs.PlateCarree())
+
+    else:
+        ax = fig.add_subplot()
+        cs = plt.scatter(longitude, latitude, c=data,
+                         s=plotopts['marker size'], vmin=plotopts['vmin'],
+                         vmax=plotopts['vmax'], cmap=plotopts['cmap'])
+        plt.grid(plotopts['grid'])
+
+    cax = fig.add_axes([ax.get_position().x1 + 0.02,
+                        ax.get_position().y0, 0.025,
+                        ax.get_position().height])
+
+    cb = plt.colorbar(cs, extend='both', cax=cax)
+    cb.set_label(plotopts['cbar label'])
+
+    plt.title(plotopts['title'], loc='left')
```

Comment:
This is wrong now (I think) because it puts the title on the colorbar axis. Maybe try just calling these before the colorbar? Or can you do ax.title?

Reply:
Yes you're right. I made changes in a notebook and forgot to change over. (along with the domains.get_domain())